apps/engineeringblog/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,103 @@
+// Addapted from https://amirardalan.com/blog/syntax-highlight-code-in-markdown
+import { FC } from "react";
+import ReactMarkdown, { Components } from "react-markdown";
+import { PrismLight as SyntaxHighlighter } from "react-syntax-highlighter";
```

Comment:
Any reason this app and `RoboShield` are using different code components/functionality?

Reply:
@kilemensi Yes. For `Roboshield` we had only the code. In this case, the `code` is embedded inside `Markdown`. We need to tell the `ReactMarkdown` how to render code.

Reply:
Cool... but my question goes both ways i.e. can the component used here also work `RoboShield`? Remember we have to maintain this code so the fewer external dependencies, the better for us.

Reply:
No @kilemensi it can't. For `Roboshield` we need to have an editor that users can edit the code if they need to. For this, we need to format how it is displayed.

Reply:
🤔 ... Assuming you're using react-syntax-highlighter for this, isn't this their [demo](https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/prism.html) showing code editing?

... anyways, lets create an issue to revisit this. Seems like we should be able to find a single package we can use and standardise on rather than pulling in a new package everytime.

Reply:
Created #879 to handle this